src/app/app.component.ts

```diff
@@ -258,6 +278,25 @@ export class AppComponent implements OnInit {
     return promise;
   }
 
+  fetchMockedTranslationsData () {
+    const promise = new Promise(function(resolve, reject) {
+      setTimeout(function() {
+        const translationsData = {
+          "form": "d8e47fbe-3ea1-3271-a9de-23c5c4c67a82",
+          "en": {
+            "Reason For Attendance": "Reason For Attendance in English"
+          },
+          "fr": {
+            "Place of Consultation:": "Place of Consultation in French:",
+            "Consultation Details": "Consultation Details in French"
+          }
```

Comment:
Is this what the translation files are supposed to look like?

Reply:
(Generally, to make things easier for translators, it's best to have one language-per-file, even if we later concatenate them via some process since most translators will work on one language or at least one language at a time.